server/api/sample_sizes.py

```diff
@@ -14,8 +14,51 @@
     suite,
 )
 from . import rounds  # pylint: disable=cyclic-import
-from .cvrs import validate_uploaded_cvrs
-from .ballot_manifest import validate_uploaded_manifests
+from .cvrs import validate_uploaded_cvrs, hybrid_contest_choice_vote_counts
+from .ballot_manifest import (
+    validate_all_manifests_uploaded,
+    hybrid_contest_total_ballots,
+)
+
+
+def validate_hybrid_manifests_and_cvrs(contest: Contest):
+    total_manifest_ballots = sum(
+        jurisdiction.manifest_num_ballots or 0 for jurisdiction in contest.jurisdictions
+    )
+    total_votes = sum(choice.num_votes for choice in contest.choices)
+    assert contest.votes_allowed is not None
+    if total_votes > total_manifest_ballots * contest.votes_allowed:
+        raise Conflict(
+            f"Contest {contest.name} vote counts add up to {total_votes},"
+            f" which is more than the total number of ballots across all jurisdiction manifests ({total_manifest_ballots})"
+            f" times the number of votes allowed ({contest.votes_allowed})"
+        )
+
+    manifest_ballots = hybrid_contest_total_ballots(contest)
+    cvr_ballots = (
+        CvrBallot.query.join(Batch)
+        .join(Jurisdiction)
+        .join(Jurisdiction.contests)
+        .filter_by(id=contest.id)
+        .count()
+    )
+    if manifest_ballots.cvr < cvr_ballots:
```

Comment:
I wonder if this should be an exact equality, since in principle these numbers should match. If there are more CVR ballots in the manifest than the CVR, maybe we throw a warning and throw an error when it's less than. @mcchilders what do you think?